src/auditlog/diff.py

```diff
@@ -131,11 +134,14 @@ def model_instance_diff(old, new):
         fields = filtered_fields
 
     for field in fields:
-        old_value = get_field_value(old, field)
-        new_value = get_field_value(new, field)
+        old_value, old_value_string = get_field_value(old, field)
+        new_value, new_value_string = get_field_value(new, field)
 
         if old_value != new_value:
-            diff[field.name] = (smart_text(old_value), smart_text(new_value))
+            if old_value is None and new_value_string=="":
```

Comment:
nit: `if old_value is None and new_value_string == ''` for pep8 style guidelines